services/IoT/applications/libraries/python.md

```diff
@@ -604,33 +599,33 @@ The *retrieveDevices()* method can be used to retrieve all the devices of a part
      print("Retrieved Devices = ", apiCli.retrieveDevices(deviceTypeId))
 ```
 
-The response contains the parameters and application that needed to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make further calls. For example, the *_bookmark* element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark returned in the response and provide it on the request for the next page. Repeat until the end of the result set indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, or the results are undefined.
+The response contains the parameters and application that are needed to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make more calls. For example, the *_bookmark* element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark that is returned in the response and provide it on the request for the next page. Repeat until the end of the result set, which is indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, or the results are undefined.
```

Comment:
"to get the array of devices returned"
does this mean "to return the array of devices" or "to get the array of devices that is returned"